### ISSUE TITLE
bug: update option active when offline 

### ISSUE THREAD

**Issue by ILoveOpenSourceApplications** [revanced/revanced-manager#986]
### Type

Error at runtime

### Bug description

When the user is offline and opens up the manager, for some reason the update option is clickable which shows whatever latest version and when pressed, proceeds to download if the user connects back in after opening it offline.

### Steps to reproduce

1) Go offline.
2) Open ReVanced Manager.
3) See that the update button is clickable.
4) Connect back in and you're able to download whatever latest version available.

### Android version

Android 13

### Manager version

1.3.8

### Target package name

ReVanced Manager 

### Target package version.

Any

### Installation type

Non-root

### Patches selected.

Nil

### Device logs (exported using Manager settings).

```shell
Nil
```

### Screenshots or videos

When offline:
![Screenshot_20230701-201707](https://github.com/revanced/revanced-manager/assets/117499019/9f17379c-bc6f-4bc1-8ff3-b1758f5f5683)

Connecting back in:
![Screenshot_20230701-201739](https://github.com/revanced/revanced-manager/assets/117499019/d4344536-4588-4ddd-964a-849bffe6c0a5)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] The issue is related solely to the ReVanced Manager and not related to patching errors or patches

**Comment by TheAabedKhan** [comment]
That's strange! This issue, which is the duplicate of #890, has already been fixed in #932. Also, I am not able to reproduce the issue since then. As you are able to reproduce the issue, can you please attach a screen recording of it?

**Comment by ILoveOpenSourceApplications** [comment]
You have to be offline inorder for this to occur.

**Comment by TheAabedKhan** [comment]
Will be fixed in the next release. Thank you so much for reporting the issue.

**Comment by ILoveOpenSourceApplications** [comment]
You're welcome and thanks for creating the fix for the issue as well.

**Comment by EvadeMaster** [comment]
🎉 This issue have been fixed by #987, please update to the latest version of ReVanced Manager (v1.3.9)

Feels free to reopen If you feels like the problem hasn't been resolved.

**Comment by dajotim937** [comment]
I'm not sure should I open new issue, so let me know if I should.

Related issue: Manager opens for really long time if internet connection without access to github (or whatever source you are using for checking updates).
Steps:
1. Have connection without access to internet (connect to router without ISP connection/blocked outgoing connection, unpaid mobile internet connection, etc).
2. Open manager.

Result: I guess ~10 secs of dark screen while waiting connection timeout (I think).

**Comment by ILoveOpenSourceApplications** [comment]
Yes you should, As my original issue and your issue is completely different.
